src/test/java/org/generationcp/middleware/dao/AttributeDAOTest.java

```diff
@@ -71,18 +76,19 @@ private void setupTestData() {
 		this.userDefinedFieldDao.save(attributeType2);
 		this.userDefinedFieldDao.save(attributeType3);
 		this.testAttributeTypes = Arrays.asList(attributeType1, attributeType2, attributeType3);
-		
-		final Attribute attribute1 = new Attribute(null, germplasm1.getGid(), attributeType1.getFldno(), 1, RandomStringUtils.randomAlphabetic(100), null, null, null);
-		final Attribute attribute2 = new Attribute(null, germplasm1.getGid(), attributeType2.getFldno(), 1, RandomStringUtils.randomAlphabetic(100), null, null, null);
-		final Attribute attribute3 = new Attribute(null, germplasm2.getGid(), attributeType1.getFldno(), 1, RandomStringUtils.randomAlphabetic(100), null, null, null);
-		this.attributeDao.save(attribute1);
-		this.attributeDao.save(attribute2);
-		this.attributeDao.save(attribute3);
+
+		this.attribute1 = new Attribute(null, germplasm1.getGid(), attributeType1.getFldno(), 1, RandomStringUtils.randomAlphabetic(100), null, null, null);
```

Comment:
@vmaletta use Random to generate the integer

Reply:
ok